.github/workflows/core.yml

```diff
@@ -736,5 +736,34 @@ jobs:
           rm unix/trezor-emu-core
           aws s3 sync --no-progress unix s3://data.trezor.io/dev/firmware/emu-nightly
 
+  core_upload_emu_branch:
+    name: Upload emulator binaries for the current branch
```

Comment:
Want to add `if: github.event_name != 'schedule'` so that it's not run twice on scheduled builds?

Reply:
Yes, this should be used.

Reply:
Done in the force-push